bucket/jabref.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "5.1",
+    "description": "Open-sourced, cross-platform citation and reference management software that uses BibTeX and BibLaTeX as its native formats and is therefore typically used for LaTeX",
+    "homepage": "https://www.jabref.org",
+    "license": "MIT",
+    "url": "https://github.com/JabRef/jabref/releases/download/v5.1/JabRef-5.1-portable_windows.zip",
+    "hash": "1a0aaa60c7424a356e3e288cc27218ddd5fe8e85ae295d87a561960404318666",
+    "extract_dir": "JabRef",
+    "bin": "JabRef.exe",
```

Comment:
What are available parameters?

Any of [these](https://github.com/JabRef/jabref/blob/master/src/main/java/org/jabref/cli/JabRefCLI.java) do not work

Reply:
Available parameters are: 

- Help: -h
- No-GUI mode: -n
- Import file: -i filename[,import format]
- Export file: -o filename[,export format]
- Import BibTeX: -importBibtex
- Export matching entries: -m [field]searchTerm,outputFile:file[,exportFormat]
- Fetch entries from Web: -f=FetcherName:QueryString
- Subdatabase from .aux file: -a infile[.aux],outfile[.bib] base-BibTeX-file
- Set file links: -asfl
- Regenerate keys: -g
- Export preferences: -x filename
- Import preferences: -p filename
- Reset preferences: -d key
- No files at startup: -b
- Version: -v
- Debug mode: --debug
- Display output in the console: --console

You need to use JabRef.bat for CLI (see the project's [documentation](https://docs.jabref.org/advanced/commandline).

Reply:
Then it is nonsense to expose jabref.exe instead of the bat...

Reply:
Ok. JabRef.exe was replaced with JabRef.bat. You might see a message "ERROR StatusLogger Unrecognized format specifier" in the terminal, which is a known issue (see issue [#6009](https://github.com/JabRef/jabref/issues/6009) with JabRef CLI.